### ISSUE TITLE
null-safety migration?

### ISSUE THREAD

**Issue by aconitine** [FilledStacks/sqflite_migration#8]
Hey there, 
I am one of a big fan of you!
I think I need to migrate to null-safety, and wanna be with sqflite_migration.
I know it's definetly on you list and that you ve been so busy.
I 'll be waiting here for the update to migrate :)
Thanks a lot!

**Comment by FilledStacks** [comment]
I will assign this internally and we'll start working on it today :)

**Comment by iampapagray** [comment]
Any updates so far?

**Comment by FilledStacks** [comment]
Yes. Migration is complete. I have to review the PR then push the update. I will get that done by tonight
